From b32bf370a29b2289f2ea85e19417e9930c61d597 Mon Sep 17 00:00:00 2001
From: amercader <amercadero@gmail.com>
Date: Thu, 26 Apr 2018 12:27:52 +0200
Subject: [PATCH] [#4031] Support group type in dataset pages breadcrumbs

We missed these in #4032, adds support for custom group types in dataset
pages
---
 ckan/templates/package/base.html | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/ckan/templates/package/base.html b/ckan/templates/package/base.html
index 1a2954a7b7..43622c756e 100644
--- a/ckan/templates/package/base.html
+++ b/ckan/templates/package/base.html
@@ -11,8 +11,9 @@
     {% set dataset = h.dataset_display_name(pkg) %}
     {% if pkg.organization %}
       {% set organization = h.get_translated(pkg.organization, 'title') or pkg.organization.name %}
-      <li>{% link_for _('Organizations'), controller='organization', action='index' %}</li>
-      <li>{% link_for organization|truncate(30), controller='organization', action='read', id=pkg.organization.name %}</li>
+      {% set group_type = pkg.organization.type %}
+      <li>{% link_for _('Organizations'), controller='organization', action='index', named_route=group_type + '_index' %}</li>
+      <li>{% link_for organization|truncate(30), controller='organization', action='read', id=pkg.organization.name, named_route=group_type + '_read' %}</li>
     {% else %}
       <li>{% link_for _('Datasets'), controller='package', action='search' %}</li>
     {% endif %}
